recipes/datreant/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: datreant
+  version: 1.0
+
```

Comment:
This is the way [continuum](https://groups.google.com/a/continuum.io/forum/#!topicsearchin/anaconda/author$3Ame/anaconda/0Rhp-rX-CYM) also handles namespace packages.

Then there really isn't any source for this package anyway, as this only provides a namespace to be used by other datreant packages.